code/xcvm/cosmwasm/contracts/gateway/src/error.rs

```diff
@@ -80,6 +80,10 @@ pub enum ContractError {
 	GatewayForNetworkNotFound(NetworkId),
 	#[error("Anonymous calls can do only limitet set of actions")]
 	AnonymousCallsCanDoOnlyLimitedSetOfActions,
+	#[error("Not supported runtime on this network")]
+	NotSupportedRuntimeOnThisNetwork,
```

Comment:
```suggestion
	#[error("RuntimeUnsupportedOnNetwork")]
	RuntimeUnsupportedOnNetwork,
```

This sounds better IMO.